system_config/test/etc/Test.conf

```diff
@@ -13,9 +13,11 @@ doubleTestValue = 0.66234023823
 #Testvalue for missing spaces between "=" and key and value
 spaceTestValue=6
 
+#Testvalue for reading a list of strings, note the missing comma between the both "hust"
+stringListTestValue = asdf,bla, blub , hust hust, möp
```

Comment:
formatting and you are also testing german characters ??

Reply:
intentional to test the parser